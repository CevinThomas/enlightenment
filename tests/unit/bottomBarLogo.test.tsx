import 'react-native';
import React from 'react';
import BottomBarLogo from '../../src/components/bottomBarLogo';

import renderer from 'react-test-renderer';

test('BottomBarLogo Component Renders', () => {
    const tree = renderer.create(
        <BottomBarLogo/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
