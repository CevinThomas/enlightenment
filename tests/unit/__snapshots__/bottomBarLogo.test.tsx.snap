// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBarLogo Component Renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "marginBottom": 20,
      "position": "absolute",
      "right": 0,
      "textAlign": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "overflow": "visible",
        "resizeMode": "contain",
        "width": 100,
      }
    }
  />
</View>
`;
